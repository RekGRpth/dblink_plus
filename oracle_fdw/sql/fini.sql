\! sqlplus -s system/manager@orcl @sql/fini_ora.sql
