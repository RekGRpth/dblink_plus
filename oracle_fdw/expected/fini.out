\! sqlplus -s system/manager@orcl @sql/fini_ora.sql

Session altered.


User dropped.


User dropped.


Directory dropped.

